src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1385,7 +1384,7 @@ internal void SaveAs(string path)
         {
             try
             {
-                Model.CurrentWorkspace.SaveAs(path, EngineController.LiveRunnerRuntimeCore);
+                model.SaveAs(path);
```

Comment:
I think saveAs should pass the workspace that is saved. if that's the case, then no need for API to save custom node.